.drone.yml

```diff
@@ -1,6 +1,7 @@
+---
```

Comment:
why are these changes in this PR?

Reply:
These aren't modifications I made myself, but I merged these from the develop branch to stay up to date with changes in the CI.